nextcloud_update.sh

```diff
@@ -593,7 +593,7 @@ else
     declare -Ag APPSTORAGE
     for app in $NC_APPS
     do
-        APPSTORAGE[$app]=$(nextcloud_occ_no_check config:app:get "$app" enabled)
+        APPSTORAGE[$app]=$(nextcloud_occ_no_check config:app:get "$app" enabled >/dev/null 2>&1)
```

Comment:
@enoch85 this will break the whole process. Did you wanted to apply this fix ad a different place? (around line 777?)

Reply:
Please change

Reply:
(it worked for me)

Reply:
I've reverted this